.github/workflows/prod-deployment.yaml

```diff
@@ -28,4 +28,4 @@ jobs:
             git pull origin main
             yarn install
             yarn build
-            pm2 restart zuri-main
+            pm2 restart zc_main_prod
```

Comment:
Why did you change this file?

Reply:
I didn't... It was fetched when I pulled request from GitHub all those files change I thought It was from your end but when I figured that's when I reverted the changes

Reply:
Okay then. I suggest closing this or then opening another one with clean commits so that I can merge it. Your functionality is actually working so it's these stray commits that are delaying the merging @The-Legend-Developer